dev-ruby/public_suffix/public_suffix-3.1.1-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="public_suffix.gemspec"
```

Comment:
There is no need to specify the GEMSPEC here since we install the gem and the defaults will take care of that.

Using the default will also avoid these warnings that are fatal without git installed:

 * Running install phase for ruby25 ...
fatal: not a git repository (or any parent up to mount point /var/tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /var/tmp)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

Reply:
Ok upgraded.